packages/mobile/src/components/drawer/Drawer.tsx

```diff
@@ -651,6 +658,35 @@ export const Drawer: DrawerComponent = ({
     }
   }
 
+  const renderContent = () => {
+    const Component = disableSafeAreaView ? View : SafeAreaView
+
+    return (
+      <Component
+        style={isFullscreen ? styles.fullScreenContent : styles.content}
+        onLayout={(event: LayoutChangeEvent) => {
+          if (!isFullscreen) {
+            const { height } = event.nativeEvent.layout
+            setDrawerHeight(height)
+          }
+        }}
+        {...(disableSafeAreaView
```

Comment:
might be nice to have this as a const before the return just to clean up the jsx here? also not sure if you can do `const viewProps = disableSafeAreaView ? null : {edges: ....}`